index.html

```diff
@@ -10,33 +10,42 @@
 <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;800&display=swap" rel="stylesheet">
 </head>
 <body>
-  <header class="container">
-    <section class="nav-bar">   
-      <a class="logo" href="">My Logo</a> 
-      <img class="union" src="/images/Union.png" alt="">   
-   </section> 
-  <main class="content">
+  <div class="container">
+    <header> 
+      <nav class="nav-bar"> 
+        <div>
+      <a class="logo" href="">My Logo</a>
+        </div> 
+        <div class="menu-item">
+      <img class="union" src="/images/Union.png" alt="Hamburger menu">
+        </div>
+      </nav>  
+    </header>
+    <headline>
+    <section class="content" style="font-family: Poppins' serif;">  
      <h1>I'm Shadrach<br/>
     Glad to see<br>
      you!
      </h1>
-  </main>
-  <article class="second-text">
+    </section>
+    <section class="second-text" style="font-family: 'Poppins', serif;">
      <p>I'm a software developer! I can help you<br>build a product feature or website. Look<br>
            through some of my work and<br>experience! If you like what you see and<br>have a project 
             you need coded, don't<br>hesitate to contact me.
      </p> 
-        <a class="let-connect">LET'S CONNECT </a><br>
-      <sub-article class="icons-social">
-        <a class="icon icon-twitter" href="">
-          <img src="/images/Twitter.png" alt=""></a>
-        <a class="icon icon-linkedin" href=""> 
-          <img src="/images/Linkedin.png" alt=""></a> 
-        <a class="icon icon-medium" href=""> <img src="/images/Vector4.png" alt=""/></a>
-        <a class="icon icon-github" href=""> <img src="/images/Github.png" alt=""/><a>
-        <a class="icon icon-iOS" href=""> <img src="/images/iOS.png" alt=""></a>
-      </sub-article>
-  </article>
-  </header>   
+        <a href="#" class="let-connect" style="font-family: 'Poppins', serif;">LET'S CONNECT </a>
+        <br>
+      <div class="icons-social">
+        <a class="icon icon-twitter" href="Twitter">
+          <img src="/images/Twitter.png" alt="Twitter icon"></a>
+        <a class="icon icon-linkedin" href="Linkedin"> 
+          <img src="/images/Linkedin.png" alt="Linkedin icon"></a> 
+        <a class="icon icon-medium" href="Messenger"> <img src="/images/Vector4.png" alt="Message icon"/></a>
+        <a class="icon icon-github" href="Github"> <img src="/images/Github.png" alt="Github icon"/><a>
+        <a class="icon icon-iOS" href=""> <img src="/images/iOS.png" alt="iOS icon"></a>
```

Comment:
- Please, let's address this issue as laid by the previous reviewer. This is how to use the `ul` to wrap the list items below:

```
<ul>
<li>
<a><img src=""></a>
</li>
</ul>
```